### DISCUSSION TITLE
Unable to import crl with python 3.8.1 - 64bit

### DISCUSSION THREAD

**Discussion by Yuangang0101** [pythonnet/pythonnet#1979] [Q&A]
### Environment

-   Pythonnet version: pythonnet  3.0.0.post1
-   Python version:3.8.1 64 bit
-   Operating System: Windows 11 64bit
-   .NET Runtime:

### Details
import clr

Traceback (most recent call last):
  File "d:\code\lightbox_SMU\Lightbox_oct_13.py", line 3, in <module>
    import clr
  File "C:\Users\XXX\AppData\Local\Programs\Python\Python38\lib\site-packages\clr.py", line 6, in <module>
    load()
  File "C:\Users\XXX\AppData\Local\Programs\Python\Python38\lib\site-packages\pythonnet\__init__.py", line 143, in load
    if func(b"") != 0:
  File "C:\Users\XXX\AppData\Local\Programs\Python\Python38\lib\site-packages\clr_loader\types.py", line 64, in __call__
    return self._callable(ffi.cast("void*", buf_arr), len(buf_arr))
RuntimeError: cannot call null pointer pointer from cdata 'int(*)(void *, int)'

-   Describe what you were trying to get done.

 clr can't support Python 3.8 64bit?
I had tried times still can't work, how can i address this issue。。
According to https://github.com/pythonnet/pythonnet/issues/1617
I changed Python  version from Python 3.10 to Python 3.8, Still not work.

**Comment by Yuangang0101** [comment]
.NET Runtime: 6.0.402

**Comment by Yuangang0101** [comment]
Maybe some issues with my current laptop, i will try the same setup with other laptop to check the difference..

**Comment by Yuangang0101** [comment]
still have this issue...after install all these .net program...crazy..

![图片](https://user-images.githubusercontent.com/51206039/196887800-e1c4690d-7f33-4558-9bd0-97af99e79f70.png)

**Comment by dw-choi-1105** [comment]
in python 3.8.10 
i downgraded pythonnet from 3.0.0 to 2.5.2
and then solved issue

**Reply by Brits00** [reply]
resolveu aqui, obrigado.
